modules/contact/test/tests/explicit_dynamics/highvel.i

```diff
@@ -206,66 +206,67 @@
         stiffness_damping_coefficient = 1.0e-3
         generate_output = 'stress_zz strain_zz'
     []
-    [inertia_x]
-        type = InertialForce
+    [Mass_x]
+        type = MassMatrix
         variable = disp_x
+        density = density
+        matrix_tags = 'system'
```

Comment:
I found the issue with the parallel inconsistency.
We should not be using the system matrix as the "mass" matrix.
Add 
```
[Problem]
  extra_tag_matrices = 'mass'
[]
```
to the top of the file then change 
`matrix_tags = "system"` to `matrix_tags = "mass"`.

Reply:
Hm. We use the same method in the tests for DirectCentralDifference and it does not lead to errors in parallel.
Though I can still give this a shot